### ISSUE TITLE
syntax error in genMol2.py

### ISSUE THREAD

**Issue by bannanc** [MobleyLab/off-ffcompare#44]
@dkyu and @vtlim 

There seems to be a syntactical error in genMol2.py:

I called
` python genMol2.py -h`
and got this error:
```
  File "genMOL2.py", line 59
    except RuntimeError, e: # catch specified warning or Runtime error
                       ^
SyntaxError: invalid syntax
```

@dkyu is this the same version of the script you are actually using? I'm on python3 now is this just a 2/3 difference problem?

**Comment by vtlim** [comment]
@bannanc, looks like a python version difference:
http://stackoverflow.com/questions/11285313/try-except-as-error-in-python-2-5-python-3-x

**Comment by bannanc** [comment]
yeah, I should have tagged this, I fixed it in my most recent PR and it runs on my desktop green planet so it should now be python2 and python3 compatible

**Comment by bannanc** [comment]
would be fixed by PR #47
